docs/Tutorials/VisualizationWithPython.ipynb

```diff
@@ -262,11 +262,84 @@
     "ax.set_aspect(\"equal\")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Transforming volume data\n",
+    "\n",
+    "Often you want to post-process volume data to compute derived quantities from\n",
+    "the ones the simulation has written out. You can sometime do this within tools\n",
+    "like ParaView (e.g. using ParaView's \"Calculator\" filter) if the computation is\n",
+    "simple enough and pointwise (i.e., needs no derivatives or other mesh\n",
+    "information). If you can't get what you need in ParaView, you can use `spectre\n",
+    "transform-vol`. It takes any Python function (a \"kernel\"), runs it over your\n",
+    "volume data, and writes the result back into the files.\n",
+    "\n",
+    "For example, let's add the number of grid points in each element as a field\n",
+    "that we can visualize:\n",
+    "\n",
+    "```sh\n",
+    "spectre transform-vol BbhVolume*.h5 -d VolumeData \\\n",
+    "    -k spectre.Spectral.Mesh3D:extents\n",
+    "```\n",
+    "\n",
+    "You will be prompted to select an output dataset name (hit enter to select the\n",
+    "default \"Extents\"). The result will be written back into the H5 files. If you\n",
+    "now regenerate an XDMF file for the volume data you will be able to see the\n",
+    "output in ParaView.\n",
+    "\n",
+    "The `transform-vol` tool supports arbitrary Python functions as kernels. Many\n",
+    "Python functions from the `spectre` module are useful kernels. For example,\n",
+    "here are some useful kernels:\n",
```

Comment:
What's the best way to discover available kernels? Could be useful to have a sentence somewhere here about "You can find existing kernels by..."